src/TabPane.js

```diff
@@ -27,9 +27,9 @@ const propTypes = {
   active: PropTypes.bool,
 
   /**
-   * Use animation when showing or hiding `<TabPane>`s. Use `false` to disable,
-   * `true` to enable the default `<Fade>` animation or
-   * a react-transition-group v2 `<Transition/>` component.
+   * Use animation when showing or hiding `<TabPane>`s. Defaults to `<Fade>`
+   * animation, else use `false` to disable or a react-transition-group v2
```

Comment:
as an aside we should probably scrub the version from RTG references now that it's on like version 4

Reply:
Apart from this, is this PR ready to be merged?

Reply:
Thanks for bumping this @kosmiq . I'll commit this text change. As far as tests go, I was having trouble faking the timing and behavior